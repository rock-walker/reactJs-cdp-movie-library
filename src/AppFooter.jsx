import React from 'react';

class AppFooter extends React.PureComponent {
    render(){
        return(
            <footer>
                <h3 className="App-logo">netflixroulette</h3>
            </footer>    
        )
    }
}

export default AppFooter;