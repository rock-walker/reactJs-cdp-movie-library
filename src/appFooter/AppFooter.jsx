import React from 'react';
import '../app/app.css';

class AppFooter extends React.PureComponent {
    render(){
        return(
            <footer>
                <h3 className="App-logo">netflixroulette</h3>
            </footer>    
        )
    }
}

export default AppFooter;