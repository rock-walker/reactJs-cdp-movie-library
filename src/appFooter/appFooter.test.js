import React from 'react';
import AppFooter from '../appFooter/AppFooter';
import renderer from 'react-test-renderer';

test('Display footer', () => {
    const component = renderer.create(
            <AppFooter/>,
        );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});