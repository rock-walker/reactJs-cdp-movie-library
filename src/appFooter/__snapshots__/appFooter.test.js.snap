// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display footer 1`] = `
<footer>
  <h3
    className="App-logo"
  >
    netflixroulette
  </h3>
</footer>
`;
